cmake/Options.cmake

```diff
@@ -46,6 +46,8 @@ function(set_options)
         set(DEFAULT_OPTION_INSTALL_FHS NO)
     endif (UNIX AND NOT APPLE)
 
+    option(OPTION_FORCE_COLORED_OUTPUT "Always produce ANSI-colored output (GNU/Clang only)." NO)
```

Comment:
Option values should be ON/OFF.

Reply:
That change is presumably the reason for the conflict

Reply:
Fixed!